src/pages/data-management/index.jsx

```diff
@@ -54,6 +55,13 @@ const DataManagementPage = ({ route }) => {
   }, []);
 
   useEffect(() => {
+    if (activeProject?.experiments.length > 0) {
+      // Right now we have one experiment per project, so we can just load the experiment
+      // This has to be changed when we have more than one experiment
+      const activeExperimentId = activeProject.experiments[0];
+      dispatch(loadBackendStatus(activeExperimentId));
```

Comment:
why is this needed?

Reply:
To check whether pipeline has been run for this experiment. To do this, the status of the `pipeline` run is checked. Backend status was not loaded for this module.

Reply:
actually this is not good because of at least 2 reasons.
1. this isn't awaited so you can be effectively acting on the backend status of the previous experiment
2. this is called every time you change an experiment which is a waste. 

What I suggest (but might be out of the scope of the PR) is to store the backend statuses indexed by experimentID so you just load them once and avoid the potential issue of using the wrong status. If you don't do that then at least `await` the backend status load if possible